src/src.pro

```diff
@@ -50,7 +50,7 @@ msvc:QMAKE_CXXFLAGS += -MP
 # Mac specific flags.
 macx:QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.7
 
-QT += network opengl uitools multimedia gui
+QT += network opengl uitools multimedia gui gamepad
```

Comment:
Use http://doc.qt.io/qt-5/qmake-test-function-reference.html#qthavemodule-name to include gamepad conditionally if it exists.